### DISCUSSION TITLE
Vercel AI SDK - Counting Tokens in NextJS App? (Amazon Bedrock - Claude 3.5 Sonnet)

### DISCUSSION THREAD

**Discussion by cbaird-1337** [vercel/community#7441] [Help] [answered]
### Summary

Hi all, I'm not sure if this is the best place to ask this, but I'm working on building an LLM chat feature into my app and struggling to find something similar to OpenAI's Tokenizer for counting tokens. My general idea here is pretty straightforward, wanting to track total token consumption (input + output) per user, in as close to real-time as possible. 

Thanks in advance!

### Steps to Reproduce

N/A

**Comment by pawlean** [comment, marked as answer]
Hi @cbaird-1337!

It might be worth asking in the [AI Discussions](https://github.com/vercel/ai/discussions).

**Reply by cbaird-1337** [reply]
I will do that, thanks!

[a comment by an automated account is not shown]
